torchvision/ops/ciou_loss.py

```diff
@@ -0,0 +1,84 @@
+import math
+
+import torch
+
+
+def ciou_loss(
```

Comment:
Good spot. The name of the file and the method can't be the same. Else it will create ambiguities on the imports from `torchvision.ops`. Following the example of giou, the file could be called `ciou_loss.py` and the method `complete_box_iou_loss`; I'm not terribly happy with the latter if I'm honest. Thoughts?